packages/camera/src/components/Controls/__tests__/FullScreenButton.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react-native';
+import '@testing-library/jest-dom';
```

Comment:
You can remove this common import statement from every file and put in one place when the Jest is bootstrapped...